jbpm-services/jbpm-services-api/src/main/java/org/jbpm/services/api/query/QueryResultMapper.java

```diff
@@ -97,7 +97,8 @@
     public static final String COLUMN_TASKID = "TASKID";
     public static final String COLUMN_WORKITEMID = "WORKITEMID";
     public static final String COLUMN_ORGANIZATIONAL_ENTITY = "ID";
-    
+    public static final String COLUMN_EXCLUDED_OWNER = "ENTITY_ID";
```

Comment:
this sound a bit too generic - while the name of the constant is clear the column name is not. Is this because of the underlying db column name and to avoid using aliases? or could we make it more clear...

Reply:
@mswiderski that's correct, I just kept the column name as the database column to avoid using an alias.